From 022afa9d6d3d1ae04344dc5f7bdbfaa9a211efb9 Mon Sep 17 00:00:00 2001
From: Xie Haocheng <haocheng.xie@amd.com>
Date: Fri, 4 Nov 2022 16:37:34 +0800
Subject: [PATCH] add CONFIG_PERF_EVENTS_AMD_BRS=y to openeuler_defconfig
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 022afa9d6d3d1ae04344dc5f7bdbfaa9a211efb9
Modified-by-SEL: No


amd inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I5ZIMJ
CVE: NA

--------------------------------

Enable AMD perf branch sample feauture in x86 openeuler defconfig.
It's implemented as a bool type option, must set 'y' to enable it.

Signed-off-by: Xie Haocheng <haocheng.xie@amd.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/x86/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 228a657e4fa8..2e354372d204 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -397,6 +397,7 @@ CONFIG_PERF_EVENTS_INTEL_UNCORE=m
 CONFIG_PERF_EVENTS_INTEL_RAPL=m
 CONFIG_PERF_EVENTS_INTEL_CSTATE=m
 CONFIG_PERF_EVENTS_AMD_POWER=m
+CONFIG_PERF_EVENTS_AMD_BRS=y
 # end of Performance monitoring
 
 CONFIG_X86_16BIT=y
-- 
2.33.0

